src/openrct2/drawing/Drawing.h

```diff
@@ -13,6 +13,7 @@
 #include "../common.h"
 #include "../interface/Colour.h"
 #include "../interface/ZoomLevel.hpp"
+#include "../world/Location.hpp"
```

Comment:
Forward declare instead (we might need to wait for the other PR so you can rebase and not do it twice)